apps/src/code-studio/components/progress/MiniViewTopRow.jsx

```diff
@@ -39,15 +39,17 @@ export default class MiniViewTopRow extends React.Component {
     scriptName: PropTypes.string.isRequired,
     // May not have this (i.e if not logged in)
     linesOfCodeText: PropTypes.string,
+    selectedSectionId: PropTypes.string,
   };
 
   render() {
-    const { scriptName, linesOfCodeText } = this.props;
-    return (
+    const { scriptName, linesOfCodeText, selectedSectionId } = this.props;
+      const queryParams = selectedSectionId ? `?section_id=${selectedSectionId}` : '';
```

Comment:
Could use this util method. https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/utils.js#L708

Reply:
Done